docs/user/get-started.md

```diff
@@ -0,0 +1,147 @@
+# Get Started
+
+For who want to use or be interested in vald, this article shows way to deploy and run vald using scylla for backup and fashion-minist for dataset.
+Also, you can see basic info of vald.
+
+1. [About](#About)
+    1. [Main Features](#Main-Features)
+    2. [Requirements](#Requirements)
+2. [Starting Vald on k8s cluster](#Starting-Vald-on-k8s-cluster)
+    1. [Deploy](#Deploy)
+    2. [Run](#Run)
+3. [Advanced](#Advanced)
+
+## About
+
+Vald is distibuted high scalable & high-speed approximate nearest neighor seardch engine.<br>
+It provides search result lists for input (e.g. word, sentence, image ...) which is coverted as (also high-dimensional) vector when searching neighor.<br>
+Vald is designed as Cloud Native and also use World Fastest ANN Algorithm [NGT](https://github.com/yahoojapan/NGT) for search neighors.
+
+### Main Features
+
+- Auto Indexing
+    - ???
+
+- Ingress Filltering
+    - ???
+
+- Horizonal Scaling
+    - Vald, which is designed as Cloud Native search engine, can use cluster resource especially memory and cpu.
+
+- Auto Index Backup
+    - Vald has auto index backup with MySQL + Redis or Cassndora which is helpful for quick recovery.
+
+- Distributed Indexing
+    - Vald indexes insert vecotr to distibuted multiple NGT agent. 
+
+- Index Replication
+    - ???
+
+- Easy to use
+    - Vald can be easily installed in a few steps and accepts many kind of settings.
+
+### Requirements
+
+- k8s: 
+- go:
+- docker:
+- helm: 
+
+If helm is not installed, let see below details or [here](https://github.com/helm/helm#install).
+
+<details>
+    <summary>optional installation</summray>
+    <div>
+    <pre>
+        1. helm
+            <code>
+            # brew|Scoop|GoFish
+            [brew|scoop|gofish] install helm
+            # chocolatey
+            choco install kubernetes-helm
+            </code>
+    </pre>
+    </div>
+</details>
+
+## Starting Vald on k8s cluster
+
+You have to install helm. Please check [here](https://github.com/helm/helm#install).
+
+### Deploy
+
+There shows procudure of deployment for vald with scylla, which is used for backup database.
+If you want to learn about scylla more, please refer to [here](https://www.scylladb.com/).
+Please confirm `kubectl cluster-info` before deployment.
+
+1. Prepare scylla database and k8s metrics-server
+
+    At first, you have to apply scylla as backup database.
+
+    ```bash
+    cd {vald_project_root}
+    
+    kubectl apply -f k8s/jobs/db/casandora/configmap.yaml
```

Comment:
I think the command below is correct
`kubectl apply -f k8s/jobs/db/initialize/cassandra/configmap.yaml`